netbox_routing/forms/bulk_import/eigrp.py

```diff
@@ -18,34 +20,56 @@
 
 class EIGRPRouterImportForm(NetBoxModelImportForm):
     device = CSVModelChoiceField(
-        queryset=Interface.objects.all(),
-        required=False,
+        queryset=Device.objects.all(),
+        required=True,
         to_field_name='name',
         help_text=_('Name of device')
     )
-
+    
     class Meta:
         model = EIGRPRouter
         fields = ('device', 'rid', 'mode', 'name', 'pid', 'description', 'comments', 'tags',)
 
 
 class EIGRPAddressFamilyImportForm(NetBoxModelImportForm):
-
+    router = CSVModelChoiceField(
+        queryset=EIGRPRouter.objects.all(),
+        required=True,
+        to_field_name='name',
+        help_text=_('Name of device')
+    )
+    
+    vrf = CSVModelChoiceField(
+        queryset=VRF.objects.all(),
+        required=False,
+        to_field_name='name',
+        help_text=_('Name of VRF (if applicable)')
+    )
+    
+    # family = CSVModelChoiceField(
+    #     queryset=IPAddressFamilyChoices.objects.all(),
+    #     required=True,
+    #     to_field_name='name',
+    #     help_text=_('IPv4/IPv6 Address Family')
+    # )
+    
     class Meta:
         model = EIGRPAddressFamily
-        fields = ('router', 'family', 'rid', 'description', 'comments', 'tags',)
+        fields = ('router', 'vrf', 'family', 'rid', 'description', 'comments', 'tags',)
 
 
 class EIGRPNetworkImportForm(NetBoxModelImportForm):
     router = CSVModelChoiceField(
         queryset=EIGRPRouter.objects.all(),
         required=True,
+        to_field_name='name',
         help_text=_('PK of Router Instance')
     )
     address_family = CSVModelChoiceField(
         queryset=EIGRPAddressFamily.objects.all(),
         required=False,
-        help_text=_('PK of Address Family')
+        to_field_name='id',
```

Comment:
```suggestion
```

Not required as PK/ID are the same and it is considered the default on a relationship